src/api/app/controllers/public_controller.rb

```diff
@@ -153,6 +153,13 @@ def distributions
     render 'distributions/index'
   end
 
+  # GET /public/request/:number
+  def show_request
+    required_parameters :number
+    req = BsRequest.find_by_number!(params[:number])
```

Comment:
Everything handled by the default handlers, but I added a test case for this here: ce83ef4ad3ebaa592020468e79a39377b3a98360